static/src/components/setting/index.vue

```diff
@@ -352,6 +353,7 @@ export default {
                 const projectData = await projectRes.json();
                 if (projectData['status'] !== 'success') {
                     validConfig = false;
+                    errormsg = projectData['error_msg'];
```

Comment:
Would it be safer to use: `projectData.get('error_msg')`

Reply:
In JavaScript `.get()` is not a method like it is in a python `dict`. If the value does not exist, an exception would not be thrown.